CHANGELOG.rst

```diff
@@ -35,6 +35,10 @@
 - The ``simulate`` function only takes one positional argument, and all other
   arguments are keyword-only.
 
+- The ``msp`` CLI has been stripped of all sub-commands except for
+  ``simulate`` and ``mutate``. All removed sub-commands except for ``newick``
```

Comment:
This isn't quite right I think. How about "The functionality of these old sub-commands is now provided by tskit, either through the ``tskit`` CLI or API functions.".